orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/model/Stage.java

```diff
@@ -422,6 +422,44 @@ public Task taskById(String taskId) {
     }
   }
 
+  /**
+   * Get all stages that are descendants of the current one
+   */
+  public List<Stage> descendants() {
```

Comment:
What do you think about `descendants()` alongside the existing `downstreamStages` methods?

The only thing I could think of would be calling it `allDownstreamStages()` or something to indicate similar intentions.

Not a strong opinion on this, and it's something that _could_ be refactored later.

Reply:
yeah, i think that would be more consistent esp since we have `ancestors` + `ancestorsOnly` (though it's an internal helper method). Anyway, I like, changing